internal/pkg/override/patch.go

```diff
@@ -14,6 +15,18 @@ import (
 
 const jsonPointerSeparator = "/"
 
+func ScaffoldWithPatch(fs afero.Fs, dir string) error {
```

Comment:
comment pls

Reply:
there's gotta be some linter i can enable for this